File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/riyousha/SaveRiyoushaManagerCapsuleDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.riyousha;

import java.io.Serializable;

import net.seijishikin.jp.normalize.common_tool.dto.DtoEntityInitialValueInterface;
import net.seijishikin.jp.normalize.common_tool.dto.FrameworkCapsuleDto;

/**
 * 利用者運営者保存Dto
 */
public class SaveRiyoushaManagerCapsuleDto extends FrameworkCapsuleDto // NOPMD DataClass
        implements Serializable, DtoEntityInitialValueInterface {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 利用者運営者Dto */
    private RiyoushaManagerDto riyoushaManagerDto = new RiyoushaManagerDto();

    /**
     * 利用者運営者Dtoを取得する
     * 
     * @return 利用者運営者Dto
     */
    public RiyoushaManagerDto getRiyoushaManagerDto() {
        return riyoushaManagerDto;
    }

    /**
     * 利用者運営者Dtoを設定する
     * 
     * @param riyoushaManagerDto 利用者運営者Dto
     */
    public void setRiyoushaManagerDto(final RiyoushaManagerDto riyoushaManagerDto) {
        this.riyoushaManagerDto = riyoushaManagerDto;
    }

}

File: app/front/src/components/main/dto/riyousha/saveRiyoushaManagerCapsuleDto.ts
import { FrameworkCapsuleDto, type FrameworkCapsuleDtoInterface } from "seijishikin-jp-normalize_common-tool";
import { RiyoushaManagerDto, type RiyoushaManagerDtoInterface } from "./riyoushaManagerDto";

interface SaveRiyoushaManagerCapsuleDtoInterface extends FrameworkCapsuleDtoInterface {

    /** 利用者運営者Dto */
    riyoushaManagerDto: RiyoushaManagerDtoInterface;
}


class SaveRiyoushaManagerCapsuleDto extends FrameworkCapsuleDto
    implements SaveRiyoushaManagerCapsuleDtoInterface {

    /** 利用者運営者Dto */
    riyoushaManagerDto: RiyoushaManagerDtoInterface;

    constructor() {
        super();

        this.riyoushaManagerDto = new RiyoushaManagerDto();
    }
}

export { type SaveRiyoushaManagerCapsuleDtoInterface, SaveRiyoushaManagerCapsuleDto }
